src/java/edu/ucsd/library/dams/util/EncryptedStreamNameModule.java

```diff
@@ -227,20 +228,22 @@ private String decryptStreamName( String streamName, String requestIP )
 		String[] parts = streamName.split(","); // nonce,ciphertext		
 		String argStr = decrypt(parts[0],parts[1]);
 		String[] argArr = argStr.split(" "); // ark,file,ip
-		if ( argArr == null || argArr.length != 3 )
+		if ( argArr == null || argArr.length != 5 )
```

Comment:
The arg length change doesn't line up with the noted parameters in the comment above. We should probably either remove the comment or update it?